### ISSUE TITLE
The issue of model accuracy result

### ISSUE THREAD

**Issue by Y445n9** [francescotonini/human-gaze-target-detection-transformer#5]
After training 150 epochs, following the default hyper-params in configs folders (may be the same as paper), I can't train the result as paper reported and my model AUC evaluation result is much lower. Is there anything error in my code or something? Here are my config tree and final result.
--- 
CONFIG
├── datamodule
│   └── _target_: src.datamodules.gazefollow_datamodule.GazeFollowDataModule                     
│       data_dir: /home/disk3/wx/ysc/gazefollow_extended                                         
│       batch_size: 4                                                                            
│       num_workers: 2                                                                           
│       pin_memory: true                                                                         
│       persistent_workers: true                                                                 
│       num_queries: 20                                                                          
│       gaze_heatmap_size: 64                                                                    
│       gaze_heatmap_default_value: 1                                                            
│       use_aux_heads_dataset: true                                                              
│       use_gaze_inside_only: false                                                              
│                                                                                                
├── model
│   └── _target_: src.models.hgttr_module.HGTTRLitModule                                         
│       net:                                                                                     
│         _target_: src.models.components.HGTTR.HGTTR                                       
│         num_queries: 20                                                                        
│         gaze_heatmap_size: 64                                                                  
│         aux_loss: true                                                                         
│       optimizer:                                                                               
│         _target_: torch.optim.AdamW                                                            
│         _partial_: true                                                                        
│         lr: 0.0001                                                                             
│       scheduler:                                                                               
│         _target_: torch.optim.lr_scheduler.StepLR                                              
│         _partial_: true                                                                        
│         step_size: 80                                                                          
│       criterion:                                                                               
│         _target_: src.criteria.HGTTRSetCriterion.HGTTRSetCriterion                             
│         matcher:                                                                               
│           _target_: src.matchers.HGTTRHungarianMatcher.HGTTRHungarianMatcher                   
│           loss_bbox_coeff: 1                                                                   
│           loss_giou_coeff: 2.5                                                                 
│           cost_class_coeff: 1                                                                  
│           cost_bbox_coeff: 2                                                                   
│           cost_gaze_watch_outside_coeff: 0                                                     
│           cost_gaze_heatmap_coeff: 2                                                           
│         losses:                                                                                
│           loss_bbox:                                                                           
│             _target_: src.criteria.losses.BoxesLoss.BoxesLoss                                  
│             loss_l1_weight: 1                                                                  
│             loss_giou_weight: 2.5                                                              
│             loss_weight: 2                                                                     
│           loss_label:                                                                          
│             _target_: src.criteria.losses.LabelLoss.LabelLoss                                  
│             loss_weight: 1                                                                     
│           loss_gaze_heatmap:                                                                   
│             _target_: src.criteria.losses.GazeHeatmapLoss.GazeHeatmapLoss                      
│             loss_weight: 2                                                                     
│             gaze_heatmap_size: 64                                                              
│       evaluation:                                                                              
│         _target_: src.criteria.HGTTREvaluation.HGTTREvaluation                                 
│         matcher:                                                                               
│           _target_: src.matchers.HGTTRHungarianMatcher.HGTTRHungarianMatcher                   
│           loss_bbox_coeff: 1                                                                   
│           loss_giou_coeff: 2.5                                                                 
│           cost_class_coeff: 1                                                                  
│           cost_bbox_coeff: 2                                                                   
│           cost_gaze_watch_outside_coeff: 0                                                     
│           cost_gaze_heatmap_coeff: 2                                                           
│         evals:                                                                                 
│           gaze_avg_distance:                                                                   
│             _target_: src.criteria.evals.GazePointAvgDistance.GazePointAvgDistance             
│             gaze_heatmap_size: 64                                                              
│           gaze_min_distance:                                                                   
│             _target_: src.criteria.evals.GazePointMinDistance.GazePointMinDistance             
│             gaze_heatmap_size: 64                                                              
│           gaze_heatmap_auc:                                                                    
│             _target_: src.criteria.evals.GazeHeatmapAUC.GazeHeatmapAUC                         
│             gaze_heatmap_size: 64                                                              
│           gaze_map:                                                                            
│             _target_: src.criteria.evals.GazeMAP.GazeMAP                                       
│             gaze_heatmap_size: 64                                                              
│             iou_threshold: 0.5                                                                 
│             l2_threshold: 0.15                                                                 
│       n_of_images_to_log: 4         

--- 
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃         Test metric         ┃        DataLoader 0         ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│    val/gaze_heatmap_auc     │     0.8056565523147583      │
│        val/gaze_map         │     0.17586500942707062     │
│ val/gaze_point_avg_distance │     0.23805715143680573     │
│ val/gaze_point_min_distance │     0.1677362620830536      │
│          val/loss           │      8.134025573730469      │
│        val/loss_bbox        │     0.9516591429710388      │
│       val/loss_bbox_0       │     0.9547505378723145      │
│       val/loss_bbox_1       │     0.9613879919052124      │
│       val/loss_bbox_2       │     0.9522721767425537      │
│       val/loss_bbox_3       │     0.9496158361434937      │
│       val/loss_bbox_4       │     0.9495354890823364      │
│    val/loss_gaze_heatmap    │    0.017916006967425346     │
│   val/loss_gaze_heatmap_0   │    0.018046194687485695     │
│   val/loss_gaze_heatmap_1   │    0.017840508371591568     │
│   val/loss_gaze_heatmap_2   │    0.017914146184921265     │
│   val/loss_gaze_heatmap_3   │     0.01792311854660511     │
│   val/loss_gaze_heatmap_4   │     0.0178730059415102      │
│       val/loss_label        │     0.3944806158542633      │
│      val/loss_label_0       │     0.3715522587299347      │
│      val/loss_label_1       │     0.39523109793663025     │
│      val/loss_label_2       │     0.3757466673851013      │
│      val/loss_label_3       │     0.38013216853141785     │
│      val/loss_label_4       │     0.39223966002464294     │
└─────────────────────────────┴─────────────────────────────┘
---
the first epoch AUC result is only about 0.64, can you me some advise? Thanks

**Comment by francescotonini** [comment]
Hi! If you still have this issue, have you tried the suggestion of #4?
